src/class/BasicHTTPRequest/BasicHTTPRequest.hpp

```diff
@@ -64,7 +64,7 @@ class BasicHTTPRequest
 	std::string toString(void) const;
 
   private:
-	const std::string _raw;
+	std::string _raw;
```

Comment:
where do you modify this? why remove const?

Reply:
if we leave it const, it is not copied to cgiManager